instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/types.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Dict
+
+from opentelemetry.trace import SpanKind
+
+BotoClientT = "botocore.client.BaseClient"
+
+OperationParamsT = Dict[str, Any]
+
+
+class AwsSdkCallContext:
```

Comment:
Are users supposed to use these objects directly in code or is this strictly for internal instrumentation use? For the later, I'd suggest o make the private with leading underscore.

Reply:
currently they are only (intended to be) used internally by the instrumentation. maybe in the future, some could be made available for public use, e.g. by request/response hooks.
i'll make them private for now as suggested